dask_planner/src/sql/logical.rs

```diff
@@ -368,6 +368,7 @@ impl PyLogicalPlan {
                     "Extension"
                 }
             }
+            LogicalPlan::Unnest(_unnest) => "UnNest",
```

Comment:
I don't think it makes a difference, but for consistency with DF the capitalization is `Unnest`:

```suggestion
            LogicalPlan::Unnest(_unnest) => "Unnest",
```